[2023-03-23 15:32:39,189] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.attack1 manual__2023-03-23T15:32:32.924727+00:00 [queued]>
[2023-03-23 15:32:39,197] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.attack1 manual__2023-03-23T15:32:32.924727+00:00 [queued]>
[2023-03-23 15:32:39,198] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-23 15:32:39,198] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-03-23 15:32:39,198] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-23 15:32:39,210] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): attack1> on 2023-03-23 15:32:32.924727+00:00
[2023-03-23 15:32:39,215] {standard_task_runner.py:52} INFO - Started process 12489 to run task
[2023-03-23 15:32:39,218] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'multi_attack_dag', 'attack1', 'manual__2023-03-23T15:32:32.924727+00:00', '--job-id', '3163', '--raw', '--subdir', 'DAGS_FOLDER/attack_dag.py', '--cfg-path', '/tmp/tmp3jichtnh', '--error-file', '/tmp/tmpywry1jw7']
[2023-03-23 15:32:39,219] {standard_task_runner.py:80} INFO - Job 3163: Subtask attack1
[2023-03-23 15:32:39,270] {task_command.py:369} INFO - Running <TaskInstance: multi_attack_dag.attack1 manual__2023-03-23T15:32:32.924727+00:00 [running]> on host 6447f1dacd32
[2023-03-23 15:32:39,333] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=multi_attack_dag
AIRFLOW_CTX_TASK_ID=attack1
AIRFLOW_CTX_EXECUTION_DATE=2023-03-23T15:32:32.924727+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-23T15:32:32.924727+00:00
[2023-03-23 15:32:39,348] {python.py:173} INFO - Done. Returned value was: None
[2023-03-23 15:32:39,359] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=multi_attack_dag, task_id=attack1, execution_date=20230323T153232, start_date=20230323T153239, end_date=20230323T153239
[2023-03-23 15:32:39,392] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-23 15:32:39,430] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
